dev-lang/icon/icon-9.5.1-r3.ebuild

```diff
@@ -0,0 +1,134 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic toolchain-funcs
+
+DESCRIPTION="very high level language"
+HOMEPAGE="http://www.cs.arizona.edu/icon/"
+
+MY_PV=${PV//./}
+SRC_URI="http://www.cs.arizona.edu/icon/ftp/packages/unix/icon-v${MY_PV}src.tgz"
```

Comment:
```
>>> Emerging (1 of 1) dev-lang/icon-9.5.1-r3::testworld
>>> Downloading 'http://www.cs.arizona.edu/icon/ftp/packages/unix/icon-v951src.tgz'
--2020-08-05 11:25:49--  http://www.cs.arizona.edu/icon/ftp/packages/unix/icon-v951src.tgz
Resolving www.cs.arizona.edu... 23.185.0.4, 2620:12a:8001::4, 2620:12a:8000::4
Connecting to www.cs.arizona.edu|23.185.0.4|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://www.cs.arizona.edu/icon/ftp/packages/unix/icon-v951src.tgz [following]
--2020-08-05 11:25:50--  https://www.cs.arizona.edu/icon/ftp/packages/unix/icon-v951src.tgz
Connecting to www.cs.arizona.edu|23.185.0.4|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://www2.cs.arizona.edu/icon/ftp/packages/unix/icon-v951src.tgz [following]
--2020-08-05 11:25:50--  https://www2.cs.arizona.edu/icon/ftp/packages/unix/icon-v951src.tgz
Resolving www2.cs.arizona.edu... 192.12.69.4
Connecting to www2.cs.arizona.edu|192.12.69.4|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2020-08-05 11:25:52 ERROR 404: Not Found.

!!! Couldn't download 'icon-v951src.tgz'. Aborting.
 * Fetch failed for 'dev-lang/icon-9.5.1-r3', Log file:
 *  '/var/tmp/portage/dev-lang/icon-9.5.1-r3/temp/build.log'

>>> Failed to emerge dev-lang/icon-9.5.1-r3
```
:(

Reply:
`https://` should also work here.

Reply:
Oh.. upstream changed the url for the 9.5.1 source file.. (with their change to using github for future changes).  I will update ebuild for -r3.

Reply:
Actually this might be my bad, I have a feeling I didn't turn on GENTOO_MIRRORS on my test container. I'll test this properly in the morning again, and I can do the cosmetic updates if this works.